### ISSUE TITLE
conan info not working if you have a folder with the name of a profile

### ISSUE THREAD

**Issue by piponazo** [conan-io/conan#1910]
When I run `conan info . --profile XXX` it normally gives me the information of the recipe in which I am working. Example:

```
PS C:\conan\conan-ffmpeg> conan info . --profile msvc2013Release
PROJECT: WARN: config() has been deprecated. Use config_options and configure
FFmpeg/3.3.3-0@PROJECT
    ID: 6f06f9ea78677e9e93ae1dc28db12f87fff8c36a
    BuildID: None
    URL: http://ffmpeg.org/releases
    License: LGPL v2.1+
```

However I am stumbling into some problems when I run the same command in other recipe:

```
PS C:\conan\conan-pdf3d> conan info . --profile msvc2013Release
ERROR: Specified profile 'msvc2013Release' doesn't exist.
Existing profiles: .gitignore, conanfile.py, conanfile.py~, createPdf3dPackages.py, createPdf3dPackages.py~, FindPDF3D.cmake, PDF3Didxm.h, README.md, README.md~
```

I noticed that the problem is that the profile name is identical to the name of a folder that exists in the current directory:

```
PS C:\conan\conan-pdf3d> ls


    Directory: C:\conan\conan-pdf3d


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----       10/18/2017  11:08 AM                .idea
d-----        9/29/2017   3:35 PM                binaries
d-----       10/18/2017  11:05 AM                msvc2013Debug
d-----       10/18/2017  11:05 AM                msvc2013Release
d-----       10/18/2017  11:05 AM                msvc2017Debug
d-----       10/18/2017  11:05 AM                msvc2017Release
d-----       10/18/2017  10:12 AM                test_package
d-----       10/18/2017  11:09 AM                __pycache__
-a----         2/7/2017   8:16 AM            127 .gitignore
-a----       10/18/2017   8:50 AM            352 conanfile.py
-a----       10/18/2017  11:08 AM           4616 createPdf3dPackages.py
-a----       10/18/2017   6:52 AM            611 FindPDF3D.cmake
-a----       10/18/2017   9:32 AM           5133 PDF3Didxm.h
-a----        10/2/2017   2:28 PM           1054 README.md
```
This seems like a bug.

**Comment by memsharded** [comment]
Tried to reproduce it, but it seems to work here. Tried in Win10, both py2 and py3, latest conan.

**Comment by piponazo** [comment]
In my case I can always reproduce it. This is the content of the recipe:

```

**Comment by memsharded** [comment]
This is already fixed in develop branch, will be in 0.28.

**Comment by memsharded** [comment]
This should be fixed with the latest changes in develop and #1911

**Comment by memsharded** [comment]
Could you please check this in the 0.28 release? Thanks!
